index.bs

```diff
@@ -1526,6 +1526,54 @@ provide both `BigInt` and `Number` variants of an API. As an API designer,
 consider it your job to chose the correct numeric type for the feature being
 exposed.
 
+<h3 id="idl-string-types">Represent strings appropriately</h3>
+
+<!--
+    This has been discussed in a number of places.
+    When editing this section,
+    please try not to lose any of the nuance
+    distilled from the discussions in these issues:
+
+    https://github.com/w3c/css-houdini-drafts/issues/687
+    https://github.com/w3c/csswg-drafts/issues/1217
+    https://github.com/w3c/csswg-drafts/issues/3307
+    https://github.com/heycam/webidl/issues/84
+    https://github.com/w3ctag/design-reviews/issues/87
+-->
+
+When designing a web platform feature which operates on [=strings=],
+{{DOMString}} is almost always the right Web IDL [=string type=] to use.
+This is because
+most string operations don't need
+to interpret the [=code units=] inside of the string.
+There are exceptions, however, for which Web IDL provides
+{{USVString}} and {{ByteString}}. [[!INFRA]] [[!WEBIDL]]
+
+{{USVString}} is the Web IDL type that represents [=scalar value strings=].
+For strings whose most common algorithms operate on [=code points=],
```

Comment:
You want to use scalar values here or explicitly mention algorithms that cannot deal with [=surrogates=]. (You can interpret a DOMString as a sequence of code points as well.)

Reply:
Okay, I've tweaked the wording here. Please take another look, @annevk!